### ISSUE TITLE
policygen.py requires a Github remote origin url

### ISSUE THREAD

**Issue by NiteDesign** [manheim/manheim-c7n-tools#31]
policygen.py   _policy_rst process is setting gitlink by calling git_html_url()
This process fails if not using Github repo.

I'm bypassing this by not calling the git_html_url() within policygen.py
and setting: gitlink = 'not applicable'

**Comment by jantman** [comment]
@NiteDesign Yeah... sorry, this project is rather opinionated when it comes to that, and it assumes you're using github. PRs are welcome for a fix, or you're always welcome to work around that as needed...

**Comment by jantman** [comment]
I'm labeling this issue as "help wanted". We'd be happy to accept a pull request to resolve this issue, but as it's not a feature that we'd use, we won't be implementing a fix ourselves.
